lib/matplotlib/backends/backend_pgf.py

```diff
@@ -316,8 +317,14 @@ def __init__(self):
                                      stdin=subprocess.PIPE,
                                      stdout=subprocess.PIPE,
                                      cwd=self.tmpdir)
-        except OSError:
-            raise RuntimeError("Error starting process '%s'" % self.texcommand)
+        except OSError as e:
+            if e.errno == errno.ENOENT:
+                raise OSError(errno.ENOENT, "Latex command not found. "
```

Comment:
Is there a good reason to not re-raise this as an `RuntimeError`?  By changing the exception type we are breaking API (as there is probably user code which expects to catch `RuntimeError` here).